package.json

```diff
@@ -7,8 +7,8 @@
     "package": "ncc build index.js -o dist/"
   },
   "dependencies": {
-    "@actions/core": "^1.2.2",
-    "@actions/exec": "^1.0.3"
+    "@actions/core": "^1.2.6",
+    "@actions/exec": "^1.0.4"
   },
   "devDependencies": {
     "@zeit/ncc": "^0.20.5",
```

Comment:
Looks like this is deprecated too. You might want to move from this now or some time later.